app-i18n/ibus-table-latin/ibus-table-latin-1.2.0.20100305.ebuild

```diff
@@ -10,8 +10,7 @@ SRC_URI="https://storage.googleapis.com/google-code-archive-downloads/v2/code.go
 LICENSE="GPL-3"
```

Comment:
`ibus-table-latin-1.2.0.20100305/tables/compose.txt` says: `### This table under LGPL-2.1 or later`, probably should be `GPL-3 LGPL-2.1+` (COPYING says it's GPL-3)